mdaviz/mda_folder.py

```diff
@@ -110,34 +120,56 @@ def doRefresh(self):
 
     def doPlot(self, *args):
         """Slot: data field selected (for plotting) button is clicked."""
-        from .chartview import ChartView
-        from .select_fields_table_view import to_datasets
+        from .chartview import ChartViewQt
+        from .chartview import ChartViewMpl
+        from .select_fields_table_view import to_datasets_qt
+        from .select_fields_table_view import to_datasets_mpl
 
         action = args[0]
         selections = args[1]
+        print(f"doPlot called with action: {action}, args: {args}")
```

Comment:
Ebentually, refactor as logger.debug instead of print

Reply:
created a new issue #56